temporal-sdk/src/main/java/io/temporal/worker/tuning/TunerHolder.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2022 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ * Copyright (C) 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this material except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.temporal.worker.tuning;
+
+import io.temporal.common.Experimental;
+import java.util.Objects;
+import javax.annotation.Nonnull;
+
+/**
+ * Can be used to create a {@link WorkerTuner} which uses specific {@link SlotSupplier}s for each
+ * type of slot.
+ */
+@Experimental
+public class TunerHolder implements WorkerTuner {
```

Comment:
I think `SimpleWorkerTuner` or `BasicWorkerTuner` or something is clearer

Reply:
I don't really like either of these - if anything this is a bit of a more advanced option since it allows you to compose different kinds of suppliers. This is the same name as in TS too

Reply:
For me a certain type of `WorkerTuner` should also be called a tuner. I don't mind the specific pieces used, but I think it should be clear it's a tuner from the name

Reply:
Then I'd do `CompositeTuner`

Reply:
To me, that would make it seem like it's a composite of many tuners.

Another approach since this is just 3 simple getters is to not expose this class to users. Rather you can just make this package private and have a `WorkerTuner.create` or something that accepts these three values and returns a `WorkerTuner` instance (but it's really an impl of this class). There's not a ton of value in exposing this class I don't think since there's not a lot of inheritance value for how simple it is.